src/main/scala/com/fortysevendeg/android/sqlite/metadata/SQLiteDatabaseMetaDataSupport.scala

```diff
@@ -0,0 +1,210 @@
+package com.fortysevendeg.android.sqlite.metadata
+
+import java.sql.{Connection, ResultSet, DatabaseMetaData}
+
+trait SQLiteDatabaseMetaDataSupport
+  extends DatabaseMetaData {
+
+  override def allProceduresAreCallable(): Boolean = false
+
+  override def allTablesAreSelectable(): Boolean = true
+
+  override def autoCommitFailureClosesAllResultSets(): Boolean = false
+
+  override def dataDefinitionCausesTransactionCommit(): Boolean = false
+
+  override def dataDefinitionIgnoredInTransactions(): Boolean = false
+
+  override def deletesAreDetected(`type`: Int): Boolean = false
+
+  override def doesMaxRowSizeIncludeBlobs(): Boolean = false
+
+  override def generatedKeyAlwaysReturned(): Boolean = false
+
+  override def insertsAreDetected(`type`: Int): Boolean = false
+
+  override def isCatalogAtStart: Boolean = true
+
+  override def locatorsUpdateCopy(): Boolean = false
+
+  override def nullPlusNonNullIsNull(): Boolean = true
+
+  override def nullsAreSortedAtEnd: Boolean = !nullsAreSortedAtStart
+
+  override def nullsAreSortedAtStart: Boolean = true
+
+  override def nullsAreSortedHigh: Boolean = true
+
+  override def nullsAreSortedLow: Boolean = !nullsAreSortedHigh
+
+  override def othersDeletesAreVisible(`type`: Int): Boolean = false
+
+  override def othersInsertsAreVisible(`type`: Int): Boolean = false
+
+  override def othersUpdatesAreVisible(`type`: Int): Boolean = false
+
+  override def ownDeletesAreVisible(`type`: Int): Boolean = false
+
+  override def ownInsertsAreVisible(`type`: Int): Boolean = false
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [ type  is a reserved keyword don't use it as a name.](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/784801319/issues/source?bid=2442814&fileBranchId=2493741#l48)